foundations/javascript_basics/data_types_and_conditionals.md

```diff
@@ -48,6 +48,14 @@ Note: Feel free to browse the files on the left column to gain familiarity with
 
 <div class="lesson-note lesson-note--warning" markdown="1">
 
+#### Replit Fork Limit
+
+Replit now allows 3 forks at a time. If you reach the limit of 3 forks, you can delete one or more of the previous forks to create a room for the new one. To do so, go to [Home](https://replit.com/~) then find "Recent Repls" section, and delete one of the forks listed there by clicking on 3 dots in front of it.
```

Comment:
Just a few small changes
- changing "fork" to "repl" since the limit is on repls in general, not just forked ones.
- streamlining wording for the last bit, and changing the link to go to the repls page directly. Users should be able to find the specific repls and how to delete them from there.

```suggestion
Replit now limits free accounts to having 3 repls at a time. If you reach the limit of 3 repls, you can delete one or more of the previous forks to create a room for the new one. To do so, go to [your repls](https://replit.com/repls) and delete what you no longer need.
```